notebooks/py/MERFISH_Pipeline_-_U2O2_Cell_Culture_-_1_FOV.py

```diff
@@ -173,6 +173,7 @@
 # EPY: END markdown
 
 # EPY: START code
+# TODO this crop should be (x, y) = (40, 40) but it was getting eaten by kwargs
```

Comment:
Oh, I get it.  Why not just set crop_y and crop_x to 40 then?

Reply:
I made this comment. Apologies for not making it clearer. All our pipeline component constructors accepted kwargs in the past. 

In some cases, arguments had default values. In cases where the argument with a default value was refactored, it could be missed in code calling the method, because the now-wrong name will get sent into kwargs. 

That's what happened here. The MERFISH code should be cropping at 40 pixels, however our code had not been doing that -- it's relying on (0, 0) defaults here and in all the tests. 

Josh's PR removed the kwargs and so revealed this.

Reply:
https://github.com/spacetx/starfish/issues/759